plugins/modules/falconctl.py

```diff
@@ -132,32 +146,47 @@ def add_args(self, state):
                     key = k.replace("_", "-")
                     if state == "present":
                         args.append("--%s=%s" %
-                                    (key, self.params[k]))
+                                    (key, self.__list_to_string(self.params[k])))
```

Comment:
I am included to change all types of `list` to `str`. Thoughts?

Reply:
I'm not opposed. Either way, we'll still need an extra step of input validation IMO